content/citation-tags.tsv

```diff
@@ -1,2 +1,3 @@
 tag	citation
 deep_review	doi:10.1101/142760
+hornik	10.1016/0893-6080(91)90009-T
```

Comment:
```suggestion
hornik	doi:10.1016/0893-6080(91)90009-T
```

I also recommend making the tag slightly more description with a date or keyword, like `hornik1991` or `hornik-approximation`.